higher precision meter values

karlbeecken opened the pull request:
I changed the rounding of values smaller than 1.000 in `readableDistance()` to also round to fixed 2 points for both imperial and metric system to increase the precision when zoomed in really far.

mabhub reviewed (changes requested):
First: thank you for contributing! 🙏

Before merging, just a detail: tests are failing  
*(`npm run test` and at least `npm run test-leaflet-1.6.0`. See scripts in `package.json` for other test commands)*.  

Maybe is it just needing small adjustments by updating test specs.
Could you take a look and update your PR according this?

karlbeecken commented:
Hi, thanks for the feedback. I tried running the tests on my M1 MacBook and a x86_64 Ubuntu server, both failing. On the Mac, Chrome is installed (v100.0.4896.60/arm64). On the server it fails, because – obviously – Chrome is not installed. But on the Mac it pends for a while and then spits out the following meaningless error message:
```
Promise Rejection:  Error: connect ECONNREFUSED ::1:52804
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1187:16) {
  errno: -61,
  code: 'ECONNREFUSED',
  syscall: 'connect',
  address: '::1',
  port: 52804
}
```

I see Chrome starting beforehand, I also tried running things with Rosetta 2 or starting Chrome manually before starting the tests, but no difference.

karlbeecken commented:
Thank you for the fast & helpful response! The tests are now working on my machine.
I changed the tests to reflect the changes I made and also changed to show only 1 decimal after the point, as realistic no one will measure centimeters on a Leaflet map.

mdartic reviewed (approved):
Sounds good to me :-)
Thanks for your contribution @karlbeecken
